test/edit_task_command_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule ToDo.EditTaskCommandTest do
+  use ExUnit.Case
+  import ToDo.EditTaskCommand
+  doctest ToDo
+
+  setup do
+    task1 = "Be more careful when running."
+    task2 = "Don't break any more bones."
+    task3 = "Don't wake up at 04:00."
+    task_list = [ task1, task2, task3]
+    {:ok, task_list: task_list, task1: task1, task2: task2, task3: task3}
+  end
```

Comment:
No need for setup here if you only have one test. Inline the things you need in the test.